lib/IRGen/GenDecl.cpp

```diff
@@ -593,6 +593,11 @@ static void collectGlobalList(IRGenModule &IGM,
 
   std::for_each(list.begin(), list.end(),
                 [](const llvm::WeakTrackingVH &global) {
+
+    if (!(!isa<llvm::GlobalValue>(global) ||
+           !cast<llvm::GlobalValue>(global)->isDeclaration())) {
+    global->dump();
```

Comment:
This shouldn't be invoked - this is going to print the global.

Reply:
Sorry, leftover code meant for testing.